device/ospf/ospfv2/interface/auth.yaml

```diff
@@ -0,0 +1,42 @@
+components:
+  schemas:
+    Ospfv2Interface.Authentication:
+      description: |-
+        This contains OSPFv4 authentication properties.
+        Reference: https://www.rfc-editor.org/rfc/rfc2328#appendix-D
+      type: object
+      required: [auth_type]
+      properties:
+        auth_type:
+          description: |-
+            The authentication method.
+            - md5: If the authentication type is of 'md5' then 'md5_key_id' and 'md5_key' both are to be configured.
+            - password: 
+          type: string
+          x-field-uid: 1
+          x-enum:
+            md5:
+              x-field-uid: 1
+            password:
+              x-field-uid: 2
+        md5_key_id:
+          description: |-
+            The unique MD5 Key Identifier per-interface.
+          type: string
+          minLength: 0
+          maxLength: 255
+          x-field-uid: 2          
+        md5_key:
+          description: |-
+            Authentication as an MD5 key.
+          type: string
+          minLength: 0
+          maxLength: 255
+          x-field-uid: 3
+        password:
+          description: |-
+            Authentication as a clear text password.
```

Comment:
Need more details ... maybe something like : 8 byte value to be placed as a clear text password in the Authentication TLV  of OSPFv2 PDUs.  Inputs of smaller length should be left padded with 0s